src/views/Login.vue

```diff
@@ -17,6 +17,7 @@
       <div class="login-field">
         <v-text-field
           label="Password"
+          type="password"
```

Comment:
missing validation

Reply:
also really going to recommend, for the email field above to take out the placeholder text

Reply:
in fact, I think we should take out most of the common sense placeholders

Reply:
we will reorganize most of this, but we have the structure there. There should not be rules on this field because it is the login. Taking out the place holder text and what not is not part of this branch.